de-lib/src/main/java/org/iplantc/de/apps/client/presenter/hierarchies/OntologyHierarchiesPresenterImpl.java

```diff
@@ -371,4 +454,12 @@ void fireEvent(GwtEvent<?> event) {
             handlerManager.fireEvent(event);
         }
     }
+
+    List<FilteredHierarchyCallback> createFilteredHierarchyList() {
+        return Lists.newArrayList();
+    }
+
+    Promise getCallbackPromise(List<FilteredHierarchyCallback> childCallbacks) {
```

Comment:
I think the GQuery methods are meant to read a little more like natural language:
`GQuery.when(promises).done(then)`
Maybe we could rename this method to follow the same pattern? Perhaps we could rename it to something like `whenCallbackPromises`?